.github/CODEOWNERS

```diff
@@ -30,8 +30,20 @@ src/Interactive/ @dotnet/roslyn-interactive
 src/NuGet/ @dotnet/roslyn-infrastructure
 src/Scripting/ @dotnet/roslyn-interactive
 src/Setup/ @dotnet/roslyn-infrastructure
-src/Tools/ @dotnet/roslyn-infrastructure
+src/Tools/AnalyzerRunner @dotnet/roslyn-ide
+src/Tools/BuildActionTelemetryTable @dotnet/roslyn-infrastructure
+src/Tools/BuildBoss @dotnet/roslyn-infrastructure
+src/Tools/BuildValidator @dotnet/roslyn-infrastructure
+src/Tools/dotnet-format @dotnet/roslyn-infrastructure
+src/Tools/ExternalAccess @dotnet/roslyn-ide
+src/Tools/IdeBenchmarks @dotnet/roslyn-ide
+src/Tools/IdeCoreBenchmarks @dotnet/roslyn-ide
+src/Tools/ManifestGenerator @dotnet/roslyn-infrastructure
+src/Tools/PrepareTests @dotnet/roslyn-infrastructure
 src/Tools/Source/CompilerGeneratorTools/ @dotnet/roslyn-compiler
+src/Tools/Source/RunTests @dotnet/roslyn-infrastructure
+src/Tools/ @dotnet/roslyn-infrastructure
+src/Tools/ @dotnet/roslyn-infrastructure
```

Comment:
Do we still need these?

Reply:
![image](https://user-images.githubusercontent.com/2371880/185508607-00aa3d8a-31bc-4b45-affa-c2f2d1e5a315.png)

Reply:
Actually @333fred shouldn't we have left this, so that way any _new_ thing being added to tools is implicitly owned by infrastructure (and so we can then tell them to also add themselves as owners?)

Reply:
No, these files are not subtractive. That would have left things in the state they were.